pkg/services/serviceaccounts/api/token.go

```diff
@@ -160,6 +160,14 @@ func (api *ServiceAccountsAPI) CreateToken(c *models.ReqContext) response.Respon
 		}
 	}
 
+	if api.cfg.TokenExpirationDayLimit > 0 {
```

Comment:
Could you please extend the tests to cover these changes?

Reply:
read your comment now. We should cover this change by test to verify the different queries but also this setting as a whole to get a test!